docs/simplification.md

```diff
@@ -12,89 +18,83 @@ The AsyncAPI Model SDK supports simplification of:
 - [Required](#determining-the-required-properties-for-the-model)
 - [Extend](#determining-the-extend-for-the-model)
 
+<br/>
+
 ## Determining the type for the model
 
-In order to determine all the possible types a schema can be, we both infer and use existing definitions of types, however we need to define a precedence for JSON Schema keywords for in which order they types are inferred or determined.
+In order to determine all the possible types a model can be based on schema, we both infer and use existing definitions of types, however we need to define a precedence for JSON Schema keywords for in which order they types are inferred or determined.
```

Comment:
I think I understand what this sentence says, but took me a while to figure out. Would something like the following still mean the same you wanted to say?

```suggestion
In order to determine all the possible schema types a model can be based on, we both infer and use existing definitions of types. 
Precedence for JSON Schema keywords matters, meaning we need to evaluate them in a particular order.
```

Reply:
@smoya maybe the following is better?
```suggestion
In order to determine all the possible types for a model, we both infer and use existing definitions of types. 
Precedence for JSON Schema keywords needs to be predetermined to ensure consistency.
```

Reply:
@jonaslagoni Yup, looks really good to me 👍